readme.md

```diff
@@ -416,6 +417,21 @@ Therapy.
 
 [R: psycom.net](https://www.psycom.net/depression.central.borderline.html)
 
+#### [Ganser syndrome](https://en.wikipedia.org/wiki/Ganser_syndrome)
+
+> Patient responds to questions with an incorrect answer
+
+Read more: http://www.minddisorders.com/Flu-Inv/Ganser-s-syndrome.html#ixzz51XLqg6TU
```

Comment:
Read contributor guidelines. This should be a paragraph, references go after this paragraph